tests/react_modules/createclass-callsite.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+var React = require('react');
+var Hello = require('./createclass-callsite');
```

Comment:
(>ლ) Ugh, yes, sorry, it should be.

I just pushed a new commit that corrects that and another silly import mistake. Everything seems to work as I expect, except that an ES6 class with proptypes (instead of type annotations) behaves differently locally vs being imported.

But I just hastily did this so maybe there's another silly mistake, I'll try to check this more thoroughly in a few hours!

Thanks for your time.